node_modules/col-whiteboards/lib/api.js

```diff
@@ -2001,6 +2008,7 @@ var scheduleThumbnails = function() {
     Collabosphere.updateLastWhiteboardThumbnails();
 
     // Schedule the next run
+    log.info({'course': course}, 'Will run whiteboard thumbnail generator in %d seconds', THUMBNAIL_INTERVAL);
```

Comment:
@johncrossman And in fact there's no `course` either. Must have been a bad copy/paste; I'll remove that.